.github/workflows/lint-format.yml

```diff
@@ -2,12 +2,6 @@ name: Linting and formatting
 on:
   pull_request:
   workflow_dispatch:
-    inputs:
-      branch:
-        description: 'Target branch'
-        required: false
-        default: 'develop'
-        type: string
```

Comment:
Why remove? Does it work without? How to specify branch when manually triggering workflow without this?

Reply:
The branch selection is built in nowadays, I didn't realize that when I added it.